zstacklib/zstacklib/utils/sanlock.py

```
@@ -447,47 +554,105 @@ def get_lockspace(vg_uuid):
+        return []
+
+    if not host_id:
+        return meta_backup.__dict__.values()
+    elif hasattr(meta_backup, host_id):
+        return [meta_backup[host_id]]
+    return []
+
 
 @bash.in_bash
-def init_vglk_if_need(vg_uuid):
+def repair_vglk_if_corrupted(vg_uuid):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)